### PR TITLE
add polls

### PR DESCRIPTION
**Pull request by 0xldr**

### CODE REVIEW

File: governance/polls/Sky Protocol Launch Season - Token and Product Launch Parameter Proposal - September 9, 2024.md [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+**This is a binary vote.**
+- **You may vote for a single option.**
+- **You should vote for the option which you prefer.**
+- **If you would accept either option, you should vote 'Abstain'.**
+
+**This proposal will only take effect if the concurrent Atlas v2 Upgrade proposal passes.**
+
+## Review
+
+The community can vote in this poll to express support or opposition to the following changes. In summary, this vovers the following items (find attached corresponding GitHub Repos, which include README files that explain all technical details):
```
Comment (blimpa) [inline governance/polls/Sky Protocol Launch Season - Token and Product Launch Parameter Proposal - September 9, 2024.md:31]:
```suggestion
The community can vote in this poll to express support or opposition to the changes described below. In summary, this covers the following items (find attached corresponding GitHub repos, which include README files that explain all technical details):
```

File: governance/polls/Sky Protocol Launch Season - Token and Product Launch Parameter Proposal - September 9, 2024.md [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+
+* New Token contracts and corresponding token converters
+  * [USDS](https://github.com/makerdao/sdai/tree/susds) (Upgraded DAI)
+  * [SKY](https://github.com/makerdao/sky) (Upgraded MKR)
+* The introduction of the [Sky Savings Rate (sUSDS)](https://github.com/makerdao/sdai/tree/snst)
```
Comment (blimpa) [inline governance/polls/Sky Protocol Launch Season - Token and Product Launch Parameter Proposal - September 9, 2024.md:36]:
```suggestion
* The introduction of the [Sky Savings Rate (sUSDS)](https://github.com/makerdao/sdai/tree/susds)
```

File: governance/polls/Sky Protocol Launch Season - Token and Product Launch Parameter Proposal - September 9, 2024.md [hunk 3]
```diff
@@ -0,0 +1,133 @@
...
+- **You should vote for the option which you prefer.**
+- **If you would accept either option, you should vote 'Abstain'.**
+
+**This proposal will only take effect if the concurrent Atlas v2 Upgrade proposal passes.**
+
+## Review
+
+The community can vote in this poll to express support or opposition to the following changes. In summary, this vovers the following items (find attached corresponding GitHub Repos, which include README files that explain all technical details):
+
+* New Token contracts and corresponding token converters
```
Comment (blimpa) [inline governance/polls/Sky Protocol Launch Season - Token and Product Launch Parameter Proposal - September 9, 2024.md:33]:
```suggestion
* New Token contracts and corresponding token converters initializtion.
```

Comment (blimpa) [reply governance/polls/Sky Protocol Launch Season - Token and Product Launch Parameter Proposal - September 9, 2024.md:33]: I think this list is generally a bit inconsistent in form. There are:

- verbal forms ("Use USDS to get SKY").
- nominalized forms ("SBE Upgrade")
- Some of the nominalized forms are 100% noun compounds, some are not ("The introduction of STR" in contrast to "STR introduction").

Additionally, list items are missing a period at the end and capitalization is inconsistent.

File: governance/polls/Sky Protocol Launch Season - Token and Product Launch Parameter Proposal - September 9, 2024.md [hunk 4]
```diff
@@ -0,0 +1,133 @@
...
+* [Atlas V2](https://forum.makerdao.com/t/atlas-v2-upgrade-poll-request/25010)
+* [Endgame: Announcing the Token and Product Launch forum post](https://forum.makerdao.com/t/endgame-announcing-the-token-and-product-launch/25021/1)
+
+### Launch Executive Proposal
+
+* Set the Schedule Method time for the earliest spell execution: September 17, 12:00 UTC
+
+### Deployment Requirements Before Token and Product Launch Spell
+
+* Tokens:
```
Comment (blimpa) [inline governance/polls/Sky Protocol Launch Season - Token and Product Launch Parameter Proposal - September 9, 2024.md:57]: This list is inconsistent in a similar fashion.

I don't think the requirements are very clear. What does it mean that "Splitter" is required?

**Review by blimpa — CHANGES_REQUESTED** [review]

File: governance/polls/Sky Protocol Launch Season - Token and Product Launch Parameter Proposal - September 9, 2024.md [hunk 5]
```diff
@@ -0,0 +1,133 @@
...
+### Parameters
+
+* New Tokens Initialization
+  * [USDS](https://github.com/makerdao/nst)
+  * [SKY](https://github.com/makerdao/ngt)
+  * [Savings USDS (sUSDS)](https://github.com/makerdao/sdai/tree/susds)
+    * Set Sky Savings Rate to **6.25%**
+* Initialize token converters
+  * DAI <> USDS converter
+    * DAI to USDS conversion (both ways). The exchange Rate is **1:1**
```
Comment (MakerJanSky) [inline governance/polls/Sky Protocol Launch Season - Token and Product Launch Parameter Proposal - September 9, 2024.md:83]:
```suggestion
    * DAI to USDS conversion (both ways). The exchange rate is **1:1**
```

File: governance/polls/Sky Protocol Launch Season - Token and Product Launch Parameter Proposal - September 9, 2024.md [hunk 6]
```diff
@@ -0,0 +1,133 @@
...
+* Sky Token Rewards (STR) Deployment
+  * SKY (Supply USDS to get Sky Token Rewards in the form of SKY)
+    * DssVest
+      * Rewards Distribution Rate: **600,000,000 SKY**
+      * Rewards Distribution Cap: **800,000,000 SKY**
+      * Start Date relative to the spell cast time minus 7 days
+        * `vestBgn`: `block.timestamp - 7 days`
+      * End Date is set one year after Start Date defined by duration
+        * `vestTau`: **365 days**
+      * Call `distrdibute()` in VestedRewardsDistribution contract in the spell execution
```
Comment (MakerJanSky) [inline governance/polls/Sky Protocol Launch Season - Token and Product Launch Parameter Proposal - September 9, 2024.md:107]: Should this be `distribute()` ?

**Review by MakerJanSky — CHANGES_REQUESTED** [review]

File: governance/polls/Sky Protocol Launch Season - Token and Product Launch Parameter Proposal - September 9, 2024.md (hunk 6 above)
Comment (0xldr) [reply governance/polls/Sky Protocol Launch Season - Token and Product Launch Parameter Proposal - September 9, 2024.md]: fixed in 3806c4d18ffbcb53765d683b0b79f0e3913c3c8a

File: governance/polls/Sky Protocol Launch Season - Token and Product Launch Parameter Proposal - September 9, 2024.md (hunk 3 above)
Comment (0xldr) [reply governance/polls/Sky Protocol Launch Season - Token and Product Launch Parameter Proposal - September 9, 2024.md:33]: In general we have tried to keep the wording as close to the post from BA Labs as possible. This is to ensure there is no ambiguity if the text differs between the two sources.

File: governance/polls/Sky Protocol Launch Season - Token and Product Launch Parameter Proposal - September 9, 2024.md (hunk 4 above)
Comment (0xldr) [reply governance/polls/Sky Protocol Launch Season - Token and Product Launch Parameter Proposal - September 9, 2024.md:57]: See above comment

File: governance/polls/Sky Protocol Launch Season - Token and Product Launch Parameter Proposal - September 9, 2024.md (hunk 2 above)
Comment (0xldr) [reply governance/polls/Sky Protocol Launch Season - Token and Product Launch Parameter Proposal - September 9, 2024.md:36]: This is not a typo, we believe it is intentional to link to the snst branch as this is the branch and commit that was audited

File: governance/polls/Atlas v2 Upgrade - September 9, 2024.md [hunk 7]
```diff
@@ -0,0 +1,53 @@
...
+Atlas v2 may be viewed at the following links:
+- [GitHub](https://github.com/makerdao/next-gen-atlas)
+- [Powerhouse Atlas Explorer](https://sky-atlas.powerhouse.io/)
+
+Please review the discussion [thread](https://forum.makerdao.com/t/atlas-v2-upgrade-poll-request/25010) to help inform your position before voting.
+
+## Outcomes
+
+**If the votes for the 'Yes' option exceed the votes for the 'No' option then the following actions will be taken:**
+* Atlas v2 will replace The Atlas defined in MIP101.
```
[the remark this answers is not in the document]
Comment (0xldr) [reply governance/polls/Atlas v2 Upgrade - September 9, 2024.md:41]: wording improved in https://github.com/makerdao/community/pull/2732/commits/2738441ba896dec4789046d0f3d680d9ae4dd010

**Review by MakerJanSky — APPROVED** [review]
Nice work, ldr.

**Pull request merged**